docs/tempo/website/api_docs/_index.md

```diff
@@ -97,7 +97,7 @@ GET /api/traces/<traceid>
 The following query API is also provided on the querier service for _debugging_ purposes.
 
 ```
-GET /querier/api/traces/<traceid>?mode=xxxx&blockStart=0000&blockEnd=FFFF
+GET /querier/api/traces/<traceid>?mode=xxxx&blockStart=0000&blockEnd=FFFF&start=<start>&end=<end>
```

Comment:
It's good that you added these params to the querier docs (leave them), but we also need to add them to the query frontend endpoint a few lines above:

https://github.com/grafana/tempo/blob/main/docs/tempo/website/api_docs/_index.md?plain=1#L94